packages/core/src/Tooltip/index.tsx

```diff
@@ -252,30 +253,47 @@ export const Tooltip: FC<TooltipProps | ExpandedTooltipProps> = ({
   const child = Children.only(children) as ReactElement
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null)
 
-  const [visible, show, hide] = useBoolean(false)
-  const [debouncedVisible, setDebouncedVisible] = useState(visible)
+  // State for click
+  const [visibleByClick, showByClick] = useState(false)
+  // Delayed state for pointer
+  const [visibleDelayed, showDelayed, hideDelayed] = useBoolean(false)
+  // State for pointer
+  const [debouncedVisible, setDebouncedVisible] = useState(false)
   const [layout, setLayout] = useState<Placement>('down')
   const [tooltipEl, setTooltipEl] = useState<HTMLDivElement | null>(null)
 
+  // If tooltip should be shown
+  const visible = visibleByClick !== debouncedVisible
```

Comment:
Depending on what behavior is preferred an alternative could be:
`const visible = visibleByClick || debouncedVisible`